concent_api/core/tests/test_unit_validation.py

```diff
@@ -727,3 +729,16 @@ def test_that_wrong_deposit_address_raises_concent_validation_error(self):
         )
         with pytest.raises(ConcentValidationError):
             validate_subtask_results_verify(self.subtask_results_verify, different_deposit_contract_address)
+
+
+class TestSubtaskResultsRejectedValidator:
+    def test_that_no_reason_raise_validation_error(self):  # pylint: disable=no-self-use
+        subtask_result_rejected = SubtaskResultsRejected()
+        with pytest.raises(GolemMessageValidationError):
+            validate_subtask_results_rejected_reason(subtask_result_rejected, False)
+
+    def test_that_failed_resources_reason_without_force_get_task_results_failed_in_database_raise_validation_error(self):  # pylint: disable=no-self-use
+        subtask_result_rejected = SubtaskResultsRejected()
+        subtask_result_rejected.reason = SubtaskResultsRejected.REASON.ForcedResourcesFailure
```

Comment:
You can inline message declaration with reason inside :)

Reply:
Done